Assets/PatchKit/UnityEditorExtension/Editor/UI/SafeBuildAndUploadScreen.cs

```diff
@@ -75,57 +93,35 @@ private void DrawContent()
                     GUILayout.ExpandWidth(true));
             }
             EditorGUILayout.EndHorizontal();
-
-            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider);
-
             EditorGUILayout.BeginHorizontal();
             {
-                GUILayout.Label("Secret:");
+                GUILayout.Label("Platform:");
 
                 GUILayout.FlexibleSpace();
 
                 GUILayout.Label(
-                    _appSecret,
+                    _platform.ToDisplayString(),
                     EditorStyles.miniLabel,
                     GUILayout.ExpandWidth(true));
             }
             EditorGUILayout.EndHorizontal();
-
-            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider);
-
             EditorGUILayout.BeginHorizontal();
             {
-                GUILayout.Label("Platform:");
-
+                GUILayout.Label("Secret:");
                 GUILayout.FlexibleSpace();
-
+                
                 GUILayout.Label(
-                    _platform.ToDisplayString(),
+                    _appSecret,
                     EditorStyles.miniLabel,
                     GUILayout.ExpandWidth(true));
             }
             EditorGUILayout.EndHorizontal();
-
-            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider);
-
-            if (GUILayout.Button(
-                new GUIContent("Switch app"),
-                GUILayout.ExpandWidth(true)))
-            {
-                Dispatch(() => SwitchLinkedApp());
-            }
-
-            if (GUILayout.Button(
-                new GUIContent("Switch platform"),
-                GUILayout.ExpandWidth(true)))
-            {
-                Dispatch(() => SwitchPlatform());
-            }
         }
         EditorGUILayout.EndVertical();
 
         EditorGUILayout.Space();
-
+        
+        // *** Build Settings *** //

```

Comment:
Instead of marking that this section is responsible for build settings, please move the whole code to a separate method.

Reply:
ok